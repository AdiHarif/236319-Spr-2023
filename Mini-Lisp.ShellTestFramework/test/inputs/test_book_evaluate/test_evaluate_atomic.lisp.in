(evaluate-atomic '(car a b) '(a.(x y) b.T))
(evaluate-atomic '(cdr a b) '(a.(x y) b.T))
(evaluate-atomic '(atom a b) '(a.x b.T))
(evaluate-atomic '(atom a b) '(a.(x y) b.T))
(evaluate-atomic '(eq a a) '(a.x b.T))
(evaluate-atomic '(eq a b) '(a.x b.T))
(evaluate-atomic '(eq a b) '(a.x b.x))
