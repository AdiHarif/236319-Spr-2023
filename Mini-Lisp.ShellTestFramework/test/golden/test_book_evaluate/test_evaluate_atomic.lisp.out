X
(Y)
T
NIL
T
NIL
T
